<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerwin</class>
 <widget class="QWidget" name="customerwin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_4" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="UserButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>log out</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget_5" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="hotelButton">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>hotels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="arranButton">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>price</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>500</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>897</width>
        <height>498</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="content"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
